### DISCUSSION TITLE
Customizable combo announcers

### DISCUSSION THREAD

**Discussion by AllWillFreeze** [ppy/osu#22219] [Ideas]
On stable, you're able to play sounds on combo bursts, and have these set in `skin.ini` as per below:

`CustomComboBurstSounds: 20,100,200,300,400,500,600,700,800,900,1000`

These will play files `comboburst-X.mp3` at the corresponding intervals, where X = 0, and 0 is played at 20 combo as defined in the config. 1 is 100 combo, 2 is 200 combo, and so on.

As LegacyComboSplash has yet to come in, this would only be related to images, and from what I can see, not sound (am dumb if I read it wrong). Reference being #17616 

My suggestion would be to allow a separate function called "Custom Announcer" much like Dance Dance Revolution, Taiko, and plenty of other games support. With osu! tracking combos, as well as offering achievements, it would seem reasonable to add. Not to mention, Taiko has a unique combo announcing system as well that users would chase during songs. These combo announcements are present in the default Taiko game, so porting over the feature to customise this makes sense.

Users can customise this to how they want, and can simply drop the sound files into the game, then simply selecting the sound to play at what combo. This can be disabled by default, but having the ability to edit this would allow for sound queues for combo tracking, rather than taking your eyes off the gameplay area or waiting for a break to check your combo.

**Comment by bdach** [comment]
Stable parity first. The new feature can be considered afterwards.

**Reply by AllWillFreeze** [reply]
Well it's not a new feature, just separating it from originally being linked to ComboBurst sounds.
